.github/workflows/release.yaml

```diff
@@ -138,3 +138,40 @@ jobs:
           args: release --clean --config .goreleaser/${{ env.GO_RELEASER_CONFIG }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Get new chart version from the chart repo
+        id: get_chart_version
+        run: |
+          helm repo add kusionstack https://kusionstack.github.io/charts
+          helm repo update
+          version=$(helm search repo kusionstack/karpor --versions | head -n 2 | tail -n 1 | awk '{print $2}')
+          if [ -z "$version" ]; then
+            echo "Error: Unable to fetch chart version" >&2
+            exit 1
+          fi
+          echo "Current chart version is: $version")
```

Comment:
The error of the pipeline shows that there is an extra parentheses here.

https://github.com/KusionStack/karpor/actions/runs/10445384301/job/28921252425

![image](https://github.com/user-attachments/assets/4d371f21-58d4-4b89-9de2-eaa10cbe5d0b)